### ISSUE TITLE
version 0.5.0 is failing in pip install (Windows) with Python 3.4.2

### ISSUE THREAD

**Issue by rahulsaini** [s/preprocessor#13]
C:\DataScienceWorks\SocialMediaDataAnalysis>pip install tweet-preprocessor
Downloading/unpacking tweet-preprocessor
  Downloading tweet-preprocessor-0.5.0.tar.gz
  Running setup.py (path:C:\Users\Rahul\AppData\Local\Temp\pip_build_Rahul\tweet-preprocessor\setup.py) egg_info for package tweet-preprocessor
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "C:\Users\Rahul\AppData\Local\Temp\pip_build_Rahul\tweet-preprocessor\setup.py", line 6, in <module>
        long_description = f.read()
      File "C:\Python34\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x8d in position 652: character maps to <undefined>
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "C:\Users\Rahul\AppData\Local\Temp\pip_build_Rahul\tweet-preprocessor\setup.py", line 6, in <module>

    long_description = f.read()

**Comment by rahulsaini** [comment]
OK. I was able to see a resolution in setup.py for adding unicode specific and changing  
-with open('README.rst') as f:
to
+with open('README.rst'**, encoding='utf-8'**) as f:

I downloaded 0.5.0.tar.gz and did manual build
**python setup.py build**
followed by
**python setup.py install**
but instead of building a site package folder it did this
Installed c:\python34\lib\site-packages\**tweet_preprocessor-0.5.0-py3.4.egg**
Processing dependencies for tweet-preprocessor==0.5.0
Finished processing dependencies for tweet-preprocessor==0.5.0
No Errors but how do I get the site package folder for tweet-preprocessor 0.5.0  ?

**Comment by rahulsaini** [comment]
ok, first

1. I removed faulty .egg only file copied to site package using **pip uninstall tweet_preprocessor** 
2. I had to do a **pip install .** on the directory where I earlier manually built the 0.5.0 package using 
python setup.py build
followed by
python setup.py install
3. It copied all files to **preprocessor** package under site-packages as desired but somehow did not copy the **_init_.py**  file. Now everything is fine. All is resolved. Much thanks. Loved your work.
